documentation/API-design-guidelines.md

```diff
@@ -585,30 +627,36 @@ Bearing in mind that APIs are continually evolving and certain operations will n
 - Remove deprecated APIs documentation.
 - Never start using already deprecated APIs.
 
+<font size="3"><span style="color: blue"> Types of modification: </span></font>
 
-<font size="3"><span style="color: blue"> Types of modification </span></font>
+- Not all API changes have an impact on API consumers. Such changes are referred to as backward compatible changes.
+- If the API undergoes changes of this type, it shall be provided through a maintenance-release which will replace the current one.
```

Comment:
I don't understand this line. My understanding is that backward compatible changes can happen:
* as part of patch release, e.g. to correct a bug within a minor version
* between minor releases (e.g. v1.2.0 and v1.3.0), e.g. introduction of new functionality without a breaking change

I suppose that the second case is not covered by "maintenance-release"?